joint_state_broadcaster/src/joint_state_broadcaster.cpp

```diff
@@ -26,12 +26,7 @@
 #include "rclcpp/time.hpp"
 #include "std_msgs/msg/header.hpp"
 
-#include "rclcpp/version.h"
-#if RCLCPP_VERSION_GTE(29, 0, 0)
-#include "urdf/model.hpp"
-#else
 #include "urdf/model.h"
```

Comment:
since it is included in the header with all the version checks, probably better to remove it from here?

```suggestion
```

Reply:
You are totally right. I missed this one

Reply:
Yess, that's what I meant [here](https://github.com/ros-controls/ros2_controllers/pull/1473#pullrequestreview-2534902220) :)